website/pages/docs/reference/manifest.mdx

```diff
@@ -40,6 +40,19 @@ Scarb bakes in the concept of [Semantic Versioning](https://semver.org/), so mak
    Always increment the minor version if you add any new public structs, traits, fields, types, functions, methods, impls or anything else.
 4. Use version numbers with three numeric parts such as 1.0.0 rather than 1.0.
 
+### `cairo-version`
```

Comment:
I feel like we should emphasize that this field does not affect the version of compiler used by scarb in any way, as I feel like this will be a common misconception among users.